client/src/boundary/model/BoundaryGame.ts

```diff
@@ -2,12 +2,12 @@ import BoundaryBoard from "./BoundaryBoard";
 
 export default class BoundaryGame {
   readonly id: string;
-  readonly active: boolean;
+  readonly state: string;
```

Comment:
vietoj string siūlau naudoti enum arba `state: 'IN_PROGRESS' | 'SOME_OTHER' | 'ANOTHER_ONE';`

Reply:
'IN_PROGRESS' | 'SOME_OTHER' | 'ANOTHER_ONE' šitą galima išsikelti į kokį global `type`